lib/mongoid/association/embedded/embeds_many.rb

```diff
@@ -168,9 +168,9 @@ def polymorphic_inverses(other = nil)
 
         def determine_inverses(other)
           matches = relation_class.relations.values.select do |rel|
+            assoc_class = rel.relation_class rescue next
```

Comment:
I'm making an assumption here, that if the class doesn't exist, we should not consider this association as a candidate for an inverse.